src/redux/actions.ts

```diff
@@ -202,3 +202,9 @@ export const updateNotifications = (payload: NotificationsPayload) => ({
   type: actionTypes.UPDATE_NOTIFICATIONS,
   payload,
 });
+export function updateSentryProject(sentryProject: string) {
```

Comment:
Just FYI, we are moving away from redux to Jotai. If you want new state variables, use jotai please.